src/Earnings/SFA.DAS.Earnings/Application/Earnings/AcademicYearComparisonHelper.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace SFA.DAS.Earnings.Application.Earnings;
+
+public static class AcademicYearComparisonHelper
+{
+    public static bool IsEarlierThan(this short academicYear, short comparisonYear)
+    {
+        return short.Parse($"{academicYear}"[..2]) < short.Parse($"{comparisonYear}"[..2]);
```

Comment:
So then the calculation effectively becomes (for say 2021, 2122):
20.21 < 21.22

Could do, I am on the fence about which is cleaner

Reply:
Ye you are right I just did the same thing, seems crazy that it would do that to me and lose the precision of 2DPs without explicitly specifying using Math.Floor or Math.DivRem etc. Updated